packages/neo-one-server-client/src/ServerManager.ts

```diff
@@ -146,6 +183,13 @@ export class ServerManager {
       return { pid, started: false };
     }
 
+    const npmResult = await npmCheck('@neo-one/server', this.checkPath, CHECK_TIMEOUT_MS, CHECK_DELAY_SEC);
+
+    if (npmResult.success && this.compareVersions(this.serverVersion, npmResult.version)) {
+      /* tslint:disable-next-line: no-console */
+      console.log(` A newer version (${npmResult.version}) of @neo-one/server is available for download `);
```

Comment:
This shouldn't assume that logging to console will actually appear to the user. E.g. consider if we build an editor client which makes use of `ServerManager` - this console.log will never appear to the user. Instead, you should signal the fact that there's a newer version via the return value and let the client implementation decide how best to display that to the user.

Reply:
👍